packages/devtools_app/lib/src/shared/memory/adapted_heap_data.dart

```diff
@@ -62,9 +61,8 @@ class AdaptedHeapData {
   static final _uiReleaser = UiReleaser();
 
   static Future<AdaptedHeapData> fromHeapSnapshot(
-    HeapSnapshotGraph graph, {
-    required String isolateId,
-  }) async {
+    HeapSnapshotGraph graph,
+  ) async {
     final objects = <AdaptedHeapObject>[];
     for (final i in Iterable.generate(graph.objects.length)) {
```

Comment:
it looks like the graph parameter is being used here, so is this removal a false positive?

Reply:
Only the isolateId was removed here (from 2 parameters `graph and `isolateId` to only `graph` remaining).